proposals/2020-fall/Agile-SIG.md

```diff
@@ -0,0 +1,43 @@
+# Application form for starting/continuing a SIG
+
+## What is/will be the name of the SIG
+Agile SIG
+
+## Which two persons will act as SIG Lead?
+- Berend Weel
+- Niels Drost
+
+## What is the SIG’s mission?
+
+The mission of the Agile SIG is to introduce, teach and train in Agile methodologies.
+We want Agile to become common knowledge in the center and aim for engineers/MT/DT/Operations to use it when appropriate.
+This will contribute to the team work within the center and to the restructuring and scaling phase we will enter in the near future.
+In addition the SIG acs as a platform for the exhange of experiences between the different teams at the center.
```

Comment:
acts

Reply:
I'm also thinking maybe this should be part of the main mission of the agile sig, perhaps:

The mission of the Agile SIG is to introduce, discuss and teach Agile methodologies within the eScience Center

Reply:
yes, done